contrib/skopeoimage/README.md

```diff
@@ -25,9 +25,7 @@ the images live are public and can be pulled without credentials.  These contain
 resulting containers can run safely with privileges within the container.
 
 The container images are built using the latest Fedora and then Skopeo is installed into them.
-The PATH in the container images is set to the default PATH provided by Fedora.  Also, the
-ENTRYPOINT and the WORKDIR variables are not set within these container images, as such they
-default to `/`.
+ENTRYPOINT of the container is set to execute the `skopeo` binary.
```

Comment:
```suggestion
The ENTRYPOINT of the container is set to execute the `skopeo` binary.
```
Or, and I'm on the fence about noting the default values or not noting them here as they can be found elsewhere

```suggestion
The PATH in the container images is set to the default PATH provided by Fedora, and as it is not set, the WORKDIR variable defaults to `/`.  The ENTRYPOINT of the container is set to execute the `skopeo` binary.
```

Your call on which to use.  I'd go with the lengthier one, but it's a coin flip to me.

Reply:
Thanks, fixed the first part.

As for `PATH`/`WORKDIR`, the way I think about this (and the way the README documents usage), users are guided to say something like `podman run …/skopeo…:latest copy`, and not actually do anything _inside_ the container. So `PATH`/`WORKDIR` should not matter for such usage. Of course users can execute the container with `--entrypoint=/bin/sh`… and the `Containerfile`s do make some effort to make `container-storage:` work within the container itself.

So it seems to me the question is whether this document is a short introduction to using the image, or whether it is documenting our “API” commitment to have `PATH` and `WORKDIR` set that way. I could see an argument for either, I very weakly prefer brevity.